### PR TITLE
Bug 1540753 Edited out dnsName value for egress docs

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1540753

Will verify in the BZ.

### CODE REVIEW

**Comment by bmeng** [comment]
The description below the json file also contains the dnsName, it should also be removed. 

When the example above is added in a project, it allows traffic to IP range
 `1.2.3.0/24` **and domain name `www.foo.com`**, but denies access to all other
 external IP addresses. (Traffic to other pods is not affected because the policy
only applies to _external_ traffic.)

For the question in bz, just delete the dnsName is ok, do not need to add second cidrSeletor.

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -215,20 +209,16 @@ To configure egress policy:
...
-all traffic.
+that matches takes effect. If the two rules in the above example were reversed,
+then traffic would not be allowed to `1.2.3.0/24` because the `0.0.0.0/0` rule
+would be checked first, and it would match and deny all traffic.
 +
-Domain name updates are reflected within 30 minutes. In the above example,
-suppose `www.foo.com` resolved to `10.11.12.13`, but later it was changed to
-`20.21.22.23`. Then, {product-title} will take up to 30 minutes to adapt to
-these DNS updates.
+Domain name updates are reflected within 30 minutes. This includes if the values change.
```
Comment (gaurav-nelson) [inline admin_guide/managing_networking.adoc]: How about:
 - `Domain name updates (including IP address change) are reflected within 30 minutes.` OR 
 - `Domain names and IP address changes are reflected within 30 minutes.`

**Comment by gaurav-nelson** [comment]
LGTM ☯️ 
 just one suggestion though.

**Comment by bfallonf** [comment]
Thanks @bmeng I've updated to include your suggestion, and included another edit from another BZ (#10335 ) . I'll merge.

**Pull request merged**